client/my-sites/themes/collections/theme-collections-layout.tsx

```diff
@@ -26,46 +18,12 @@ export interface ThemeCollectionsLayoutProps {
 	onSeeAll: ( object: OnSeeAll ) => void;
 }
 
-const { recommended, ...randomizableThemeCollections } = THEME_COLLECTIONS;
-const collections: Array< ThemeCollectionDefinition > = [
-	recommended,
-	...( config.isEnabled( 'themes/discovery-randomize-collections' )
-		? Object.values( randomizableThemeCollections ).sort( () => Math.random() - 0.5 )
-		: Object.values( randomizableThemeCollections ) ),
-];
-
-function ThemeCollectionsPatternAssemblerCta() {
-	const selectedSiteId = useSelector( getSelectedSiteId );
-	const selectedSite = useSelector( getSelectedSite );
-	const isLoggedIn = useSelector( isUserLoggedIn );
-	const siteEditorUrl = useSelector( ( state: object ) =>
-		getSiteEditorUrl( state, selectedSiteId, {
-			canvas: 'edit',
-			assembler: '1',
-		} )
-	);
-	const onClick = () => {
-		const shouldGoToAssemblerStep = isAssemblerSupported();
-
-		recordTracksEvent( 'calypso_themeshowcase_pattern_assembler_cta_click', {
-			goes_to_assembler_step: shouldGoToAssemblerStep,
-		} );
-
-		const destinationUrl = getSiteAssemblerUrl( {
-			isLoggedIn,
-			selectedSite,
-			shouldGoToAssemblerStep,
-			siteEditorUrl,
-		} );
-		window.location.assign( destinationUrl );
-	};
-	return <PatternAssemblerCta onButtonClick={ onClick } />;
-}
+const collections: Array< ThemeCollectionDefinition > = Object.values( THEME_COLLECTIONS );
 
 function ThemeCollectionsLayout( props: ThemeCollectionsLayoutProps ) {
```

Comment:
Do we need this anymore, or can we just drop the Partner `ShowcaseThemeCollection` directly into the list?

Reply:
Speaking of `ShowcaseThemeCollection`, I have two non-blocking questions to discuss:

- Should we stop randomizing the themes inside the carousel and just sort them by curation? (This seems a no-brainer to me, since we only needed the randomization to increase the variety of similar collections on the same page.)
- Should we hide the partner themes from the Recommended/All list, and only display them in the carousel? (This is a more complex question, that we can easily answer after launch.)

Reply:
> Do we need this anymore, or can we just drop the Partner ShowcaseThemeCollection directly into the list?

We could remove it for simplicity.

> Should we stop randomizing the themes inside the carousel and just sort them by curation? (This seems a no-brainer to me, since we only needed the randomization to increase the variety of similar collections on the same page.)

No strong opinion here, but, since you seem to prefer not randomizing I'll change it.

> Should we hide the partner themes from the Recommended/All list, and only display them in the carousel? (This is a more complex question, that we can easily answer after launch.)

Let's chat about this later on.

Reply:
Created a draft issue in our board so that we can do this as a follow-up.